src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { isChapter33 } from '../helpers';
+
+export class StemEligibilityView extends React.Component {
+  onChange = property => {
+    this.props.onChange({
+      ...this.props.formData,
+      ...property,
+    });
+  };
+
+  iconClass = indication =>
+    classNames('fa', {
+      'fa-check': indication,
+      'vads-u-color--green': indication,
+      'fa-times': !indication,
+      'vads-u-color--gray-medium': !indication,
+    });
+
+  iconText = (indication, text) =>
+    indication ? `You answered yes to ${text}` : `You answered no to ${text}`;
+
+  renderCheck = (classes, title, text) => (
+    <li className="vads-u-margin-bottom--0">
+      <span className="fa-li">
+        {/* eslint-disable jsx-a11y/no-noninteractive-tabindex */}
+        <i className={classes} title={title} aria-hidden="true" tabIndex="0" />
```

Comment:
I'm not sure that this needs a `tabIndex`, but I'll let Trevor make that call.

Reply:
When I was tabbing through with screenreader it wasn't tabbing to the list or any of it's elements, it was going from title to radio buttons and skipping these items

Reply:
@zurbergram The `<i>` here is just a visual decoration. We're deliberately hiding it from screen readers with the `<aria-hidden="true"` declaration, so we don't want it to be a tab stop.

Tabbing through with a screen reader will only read out the items that can take keyboard focus (links, buttons, form inputs, etc.). To hear the list items and their text read aloud, you'll need to move via virtual cursor:

* JAWS: up arrow / down arrow in browse (default) mode. Can also press `L` to traverse by lists quickly.
* NVDA: up arrow / down arrow in browse (default) mode. Can also press `L` same as JAWS
* VoiceOver (Mac): Press `Ctrl + Opt + right arrow` to move through the document. Press `Ctrl + Opt + Cmd + X` to traverse by lists.

By removing the non-semantic icons from the screen reader's accessibility tree and adding SR-only text "You answered yes to..." or "You answered no to..." we are making the implicit relationship between the icon and the person's answer, explicit for assistive technology.

Please feel free to ping me on Slack if you have questions or want to discuss this further.

Reply:
made suggested changes from https://github.com/department-of-veterans-affairs/vets-website/pull/11714#discussion_r384071246